book/src/rfcs/foreign_field_add.md

```diff
@@ -248,7 +256,7 @@ For this gate, we need to perform 4 range checks to assert that the limbs of $a,
 | 6          | $q$               |
 | 7          | $c_0$             |
 | 8          | $c_1$             |
-| 9          | $s$               |
+| 9          |                   |
```

Comment:
I think above needs update.  There's no `c_1` anymore.

Reply:
that is true, seems like I didn't modify the rfc in the last update to ffadd